src/jit/codegenarm64.cpp

```diff
@@ -1367,7 +1367,8 @@ void                CodeGen::genEmitGSCookieCheck(bool pushReg)
     getEmitter()->emitIns_R_R(INS_cmp, EA_PTRSIZE, regGSConst, regGSValue);
 
     BasicBlock  *gsCheckBlk = genCreateTempLabel();
-    inst_JMP(genJumpKindForOper(GT_EQ, true), gsCheckBlk);
+    emitJumpKind jmpEqual = genJumpKindForOper(GT_EQ, CK_SIGNED);
```

Comment:
Shouldn't we be using CK_UNSIGNED here.  The second parameter to genJumpKindForOper() previous is 'bool isUnsigned'.  Since true is passed for second param, in its place we should be passing CK_UNSIGNED?